src/Microsoft.VisualStudio.Mac.LanguageServices.Razor/Editor/DefaultTextBufferProjectService.cs

```diff
@@ -17,6 +17,7 @@ namespace Microsoft.VisualStudio.Mac.LanguageServices.Razor.Editor
     [Export(typeof(TextBufferProjectService))]
     internal class DefaultTextBufferProjectService : TextBufferProjectService
     {
+        private const string DotNetCoreRazorCapability = "DotNetCoreRazor | AspNetCore";
```

Comment:
These are the base capabilities defined for current and old Razor in the MonoDevelop world.

Reply:
Not `DotNetCoreWeb`?

Can you provide a little more information about this?

Reply:
Haven't seen a DotNetCoreWeb. Todd was told to use these capabilities when determining "IsRazorCompatible"